circuits/cpp/barretenberg/py/src/estimation/msm.py

```diff
@@ -0,0 +1,21 @@
+class MSM:
+    def __init__(self, length):
+        self.length = length
+
+    def num_gates_native(self):
+        return 960 * self.length
+
+    def num_gates_non_native(self):
+        return 22000 + 4000 * self.length
+
+    def num_gates_eccvm(self):
+        return 3 + 2 * self.length
```

Comment:
What does 3 + 2 * length refer to here?

Reply:
Constant overhead (I thought was three gates, one to initialize accumulator, one to return the value, one to clear the accumulator, or something like this?) plus 1 gate per addition plus 1 gate per mul. Maybe there's a muladd or something? I was being conservative.